client/app/components/notification/notification.controller.js

```diff
@@ -0,0 +1,36 @@
+class NotificationController {
+  constructor(User, Authentication, $state, $injector) {
+    "ngInject";
+    Object.assign(this, {
+      User,
+      Authentication,
+      $state,
+      $injector
+    });
+  }
+
+  isVerified() {
+    return this.userdata.mail_verified;
+  }
+  sendVerification(email) {
+    const $mdToast = this.$injector.get("$mdToast");
+    this.Authentication.update().then((data) => {
+      if (data.mail_verified === false) {
```

Comment:
I think you can leave away this check. The button should not be clickable in this case, so checking again seems to be too much.